plugin/pkg/shoot/tolerationrestriction/admission.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
Seems wrong?

Reply:
why ?

Reply:
This license header looks different than in other files, and it still contains stuff from the previous license header

Reply:
I can't spot any difference in the license header of this file version:
https://github.com/gardener/gardener/blob/802318270170ee6527a6845c2b3f00381f0eacc4/plugin/pkg/shoot/tolerationrestriction/admission.go#L1-L2